src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -763,8 +747,8 @@ def setUp(self):
 
             # Ensure write permissions for the daos command log files when
             # using systemctl
-            if (self.agent_manager_class == "Systemctl" or
-                    self.server_manager_class == "Systemctl"):
+            if (self.agent_manager_class == "Systemctl"
+                    or self.server_manager_class == "Systemctl"):
```

Comment:
FYI
```suggestion
            if "Systemctl" in (self.agent_manager_class, self.server_manager_class):
```

Reply:
Technically it is not the same check, but it should work for what we are doing.

Reply:
I'm okay with either approach